modules/assignment/src/main/java/com/intuit/wasabi/assignment/AssignmentsModule.java

```diff
@@ -100,6 +111,42 @@ protected void configure() {
         LOGGER.debug("installed module: {}", AssignmentsModule.class.getSimpleName());
     }
 
+    private void bindMetadataCache(final Properties properties) {
+        Boolean metadataCacheEnabled = Boolean.parseBoolean(getProperty("metadata.cache.enabled", properties, "true"));
+        bind(Boolean.class).annotatedWith(named("AssignmentsMetadataCacheEnabled")).toInstance(metadataCacheEnabled);
+
+        if(metadataCacheEnabled) {
+            //This is a cache refresh interval, at this frequency cache will be refreshed.
+            Integer metadataCacheRefreshInterval = Integer.parseInt(getProperty("metadata.cache.refresh.interval", properties, "5"));
```

Comment:
@bargenilesh is "5" five minutes? it will be good to have a constant e.g. 5_MINUTES="5" to replace "5" to help reader to understand default is 5 minutes.